doc/glm/glm_intro.rst

```diff
@@ -15,32 +15,34 @@ A primer on BOLD-fMRI data analysis
 What is fMRI ?
 --------------
 
```

Comment:
The title at the top of this file should make it clear that it is a theoretical part.

I worry that people think that they need to read this to figure out how to use nilearn, as it comes first.